go.mod

```diff
@@ -64,7 +64,7 @@ require (
 	k8s.io/apiextensions-apiserver v0.32.0
 	k8s.io/apimachinery v0.32.0
 	k8s.io/apiserver v0.32.0
-	k8s.io/client-go v0.32.0
+	k8s.io/client-go v1.5.2
```

Comment:
This is wrong. https://pkg.go.dev/k8s.io/client-go?tab=versions says that 0.32.0 is the latest version and that 1.5.2 was retracted in 2021.

Reply:
When I run `go get k8s.io/client-go` in the top-level directory of our repo on my laptop, it also upgrades to 1.5.2. Not sure what's going on here, but we definitely don't want v1.5.2.

Reply:
It appears that v1.5.2's `go.mod` file is still retracting itself: https://github.com/kubernetes/client-go/blob/v1.5.2/go.mod#L39

Reply:
`go list -m -versions k8s.io/client-go` is only supposed to show non-retracted versions, but for this module it shows all the retracted versions too, so something is very fishy here.

Reply:
It seems to have something to do with our `replace` directives. If you temporarily remove the `replace` directives then `go get` no longer tries to upgrade us to a retracted version. We were using the `replace` directives to temporarily hold back the Kubernetes version bump due to a bug in Kubernetes v0.32.0 packages, but this strategy seems to confuse `go get` for some unexpected reason.

Reply:
This is probably a bug in `go get -u` that only manifests when a `replace` directive exists. `go get -u` will usually float to the latest, but without the `replace` directive it floats to the latest non-retracted version, and it looks like with a `replace` directive it will float to the latest version (retracted or not).